contents.json

```diff
@@ -5541,6 +5541,11 @@
       "category": "key-value-store",
       "description": "A lightweight wrapper over UserDefaults & NSUserDefaults with an extra AES-256 encryption layer.",
       "homepage": "https://github.com/vpeschenkov/SecureDefaults"
+    }, {
+      "title": "Codextended",
+      "category": "utility",
+      "description": "xtensions giving Swift's Codable API type inference super powers.",
```

Comment:
Hi @bennettyuan18, could you fix the following issues:
1. Description does not say "written in Swift" or variant (Swift's Codable)
2. It seems you forgot "E" ("xtensions" => "Extensions")

Reply:
I fixed. 👌